### ISSUE TITLE
Sprint 1.37.0 1/2 - May 30 to June 10

### ISSUE THREAD

**Issue by marcushyett-ph** [PostHog/posthog#9876]
# Global Sprint Planning

## Retro: Status of Outcomes from Previous Sprint

1. Have the plugin server split between ingestion and runner servers live on Cloud @macobo (Status: All the code is in, we aim to turn this on tomorrow - 26/05)
Why: To make the plugin server more reliable and easier to manage.
2. Eliminate all legacy ingestion code from the plugin server @tiina303 (Status: We will be done by Friday, currently blocked by one test)
Why: To reduce tech debt and make the plugin server easier to maintain.
3. Having a dev / staging / prod environments setup and running in new AWS account using EKS in parallel with existing ECS cluster. All stateful resources will be shared between the two environments (Kafka, PG, CH, Redis) but traffic will be able to be consumed from new environment @guidoiaquinti & @hazzadous (Status: see below)
why: to move towards our 6 month goals of having infrastructure as code
4. Generate a PNG/SVG/PDF export for any chart @benjackwhite  (Status: Deployed behind feature flags. We are going to focus on PNG only right now)
why: So we can have better sharing links + subscribe to dashboards
5. Start design/product work on dashboard/insight subscriptions @benjackwhite  (Status: [Designs are there](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=10306%3A42331))
why: so we can build subscribing to dashboards once technical hurdles have been cleared
6. Onboarding 1.5, increase the % of people that ingest events @liyiy (Status: Work to be wrapped up this sprint, experiment to be launched early next week.)
why: so more people can be successful with PostHog
7. Persons on events querying for trends, but goal is all insights @neilkakkar (All but lifecycle is sorted, pending a new column. 'New feature' around funnel properties & breakdown filtering rolled over to next sprint)
why: To improve performance for biggest customers
9. Async querying @rcmarron (Status: Deprioritized day-1 of sprint)
why: 4% of queries time out, this will allow those queries to complete

## Support hero this sprint

Week 1: Karl
Week 2: Marcus?

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1.
2.
3.
4.
5.

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Bugs

-

### Low priority / side quests

-

```

**Comment by yakkomajuri** [comment]
# Ingestion

## Goals

### [1. Fix the main known issues regarding reliability and data integrity (owner: @tiina303)](https://github.com/PostHog/posthog/issues/9958) 

**Why:** To make the plugin server more reliable and prevent churn from data integrity issues
**Why now:** To solve pressing customer issues and make us comfortable deploying large changes to ingestion

### [2. Complete the final ingestion-related tasks for persons/groups on events (owner: @macobo)](https://github.com/PostHog/posthog/issues/9958) 

**Why:** To unlock high scale querying
**Why now:** To ensure our ingestion pipeline guarantees hold true in its new proposed state

**Comment by benjackwhite** [comment]
Team East

## Retro



- 

## Hang over items from previous sprint



1. Generate a PNG/SVG/PDF export for any chart
- Very nearly complete. Just waiting on review to be merged. ONLY PNG currently
- Vote to leave out SVG completely
- Vote to add PDF button but with a "coming soon" response + tracking to see if anyone even wants this
2. Start design/product work on dashboard/insight subscriptions 
- Figma: https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=10306%3A42331

## Planning

### High priority

- Subscriptions - Scheduling of **email** reports for any Insight (using export functionality to generate images)
   -  (50% of team is on holiday during this sprint)

### Bugs

- https://github.com/PostHog/posthog/issues/9959
- Potentially (non-documented) bug with Insights not refreshing frequently enough - should be looked into for Subscriptions to work

### Low priority / side quests

-

**Comment by fuziontech** [comment]
# Team Platform Infra

## Retro

- Tim - Good sprint. Released TZ. Big thing he was working on. Did a few smaller things related to billing.
- Guido - Sprint went well. Surprised that we were able to get what we planned done despite some distractions. Shipped object store. Made a lot of progress. Not on track for 1% by end of the week. Miracle that we got so much done. Concern: we always work at 120% - Let's not open up can of worms RE: support hero. Stats would be good for support hero. It's hard to dig into things that you haven't touched before. 12 hour working. Not sustainable.
- Harry - It's been good. Working double duty covering Guido's shift because of support hero. Gotta pump out code.  Get merge happy.
- James - Solid sprint. We are now on 22.3. Probably could moved faster on this. I think we were really risk averse. Hiring is going well. Got involved a lot more in plugin-server work fixing rogue tests.

## Hang over items from previous sprint

- (folded into priority below) - 1% of traffic to EKS prod (Prioritized)

## Planning

### High priority

- Switch over to using EKS for the web (events and for app) @hazzadous & @guidoiaquinti 
- Move _all_ of session recording parts to ST1 @fuziontech

**Comment by EDsCODE** [comment]
Retro — Feelings TBD on Firday

Priorities:

1. Simplifying events + actions @alexkim205  — Customers struggle to differentiate or group similar events, causing inconsistency in reporting and inaccurate analyses
2. Nail FOSS @EDsCODE  —  This has been stale for several months and people can’t properly use FOSS until the refactor is done. 

Side Quests:

- Feature flags @neilkakkar 
     - Make sure distinctIDs for same person are in same bucket given rollout %. (??)
     - For experiments, makes sure variants don’t flip.
- Funnel first-touch / last-touch Event & Person properties @neilkakkar 
     - Some semantics changing with person-on-events
     - Remove limitation of ‘all events should have the event property for breakdown to work’
- Recordings @rcmarron 
     - Compress data before sending it down to player (speed up recording playback for eu customers)
- Investigate rrweb bugs (SVG issue, CSSMediaRules, CSS-grid issue)
- UI cleanup design work @liyiy @clarkus @lottiecoxon 
Squeak ping
